packages/react-components/lib/tasks/create-task.tsx

```diff
@@ -347,6 +351,23 @@ function defaultTask(): SubmitTask {
   };
 }
 
+function getToolTipMessage(errMsgs: string[]) {
+  return errMsgs.length === 0 ? (
+    ''
+  ) : (
+    <div>
+      {errMsgs.map((msg) => {
+        return <div>{msg}</div>;
+      })}
+    </div>
+  );
+}
+
+export interface SubmitTaskDetails {
+  submitTask: SubmitTask[];
+  errors: { [index: string]: string[] };
```

Comment:
type is declared as `string` but it's being accessed with `number` in several places.